crates/aptos-telemetry/Cargo.toml

```diff
@@ -38,6 +38,7 @@ aptos-metrics-core = { path = "../../crates/aptos-metrics-core" }
 
 aptosdb = { path = "../../storage/aptosdb" }
 consensus = { path = "../../consensus" }
+flate2 = "1.0"
```

Comment:
Nit: Move a few lines up to where the external dependencies are? Not sure if CI will make this change for us.